versioned_docs/version-8.7/apis-tools/java-client/job-worker.md

```diff
@@ -130,6 +130,13 @@ This implementation creates two metrics:
 - `zeebe.client.worker.job.activated`: A counter tracking the count of jobs activated.
 - `zeebe.client.worker.job.handled`: A counter tracking the count of jobs handled.
 
+:::info Deprecated metrics
+The following metrics will be deprecated from 8.8 and will be removed in version 8.10:
```

Comment:
Are we just noting in these two 8.7 docs for awareness, with more formal announcements in 8.8 docs?